plugins-extra/query_enhancements/public/plugin.ts

```diff
@@ -72,6 +80,26 @@ export class QueryEnhancementsPlugin
       },
     });
 
+    data.__enhance({
+      ui: {
+        query: {
+          language: 'SQL Async',
```

Comment:
did we verify the language with spaces has no issues?

Reply:
I do get errors when the language is `SQL Async` and `SQL_Async`, but I think they might be due to the index not being set correctly and other miscellaneous things related to the datasources type not being in this branch. Will verify